package.json

```diff
@@ -21,9 +24,9 @@
   },
   "devDependencies": {
     "@types/argv": "0.0.5",
-    "@types/node": "^16.4.10",
-    "@types/lodash": "^4.14.172",
-    "chai": "^4.3.4",
-    "mocha": "^9.0.3"
+    "@types/node": "^18.0.0",
```

Comment:
Does the `@types/node` package reflect the definitons for node 18? Also is it really something we **_need_** to include?

Reply:
I would assume so.  Do you think we should purposely not bump from 16, so no one sees any types specific to 18?  Should we restrict the engine to node 14 and 16 only?

Reply:
I guess I'm wondering what the point of updating the types definitions is and if there's a reason to require something newer than the engine. Did this trigger an audit warning or are we just upgrading it while we were upgrading mocha and chai?